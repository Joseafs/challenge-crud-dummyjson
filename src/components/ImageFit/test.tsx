import { render } from '@testing-library/react';

import { snapshotYamamotoDescription } from '~/utils/mocks/snapshotMockValues';

import { ImageFit } from '.';

const imagePublicPath = '/images/favicon.png';

describe('ImageFit', () => {
  test('Should match snapshot', () => {
    const { container } = render(
      <ImageFit description={snapshotYamamotoDescription} height={500} src={imagePublicPath} width={500} />,
    );
    expect(container).toMatchSnapshot();
  });
});
