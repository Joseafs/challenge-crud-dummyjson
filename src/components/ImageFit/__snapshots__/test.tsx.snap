// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageFit Should match snapshot 1`] = `
<div>
  <img
    alt="Yamamoto, the Captain-Commander of the Gotei 13, wields the fearsome Zanpakuto Ryujin Jakka. His fiery power and unwavering resolve make him a legend in Soul Society."
    class="sc-396cbb3d-0 fJFgPN"
    data-nimg="1"
    decoding="async"
    description="Yamamoto, the Captain-Commander of the Gotei 13, wields the fearsome Zanpakuto Ryujin Jakka. His fiery power and unwavering resolve make him a legend in Soul Society."
    height="500"
    loading="lazy"
    sizes="auto"
    src="/_next/image?url=%2Fimages%2Ffavicon.png&w=3840&q=75"
    srcset="/_next/image?url=%2Fimages%2Ffavicon.png&w=16&q=75 16w, /_next/image?url=%2Fimages%2Ffavicon.png&w=32&q=75 32w, /_next/image?url=%2Fimages%2Ffavicon.png&w=48&q=75 48w, /_next/image?url=%2Fimages%2Ffavicon.png&w=64&q=75 64w, /_next/image?url=%2Fimages%2Ffavicon.png&w=96&q=75 96w, /_next/image?url=%2Fimages%2Ffavicon.png&w=128&q=75 128w, /_next/image?url=%2Fimages%2Ffavicon.png&w=256&q=75 256w, /_next/image?url=%2Fimages%2Ffavicon.png&w=384&q=75 384w, /_next/image?url=%2Fimages%2Ffavicon.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Ffavicon.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Ffavicon.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Ffavicon.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Ffavicon.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Ffavicon.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Ffavicon.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Ffavicon.png&w=3840&q=75 3840w"
    style="color: transparent;"
    width="500"
  />
</div>
`;
